com.chartboost.mediation/Runtime/AdFormats/Banner/IChartboostMediationBannerView.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Chartboost.Banner;
+using Chartboost.Events;
+using Chartboost.Requests;
+using Chartboost.Results;
+
+namespace Chartboost.AdFormats.Banner
+{
+    public delegate void ChartboostMediationBannerEvent(IChartboostMediationBannerView bannerView);
+    public delegate void ChartboostMediationBannerDragEvent(IChartboostMediationBannerView bannerView, float x, float y);
+    
+    public enum ChartboostMediationBannerHorizontalAlignment
+    {
+        Left,
+        Center,
+        Right
+    }
+    
+    public enum ChartboostMediationBannerVerticalAlignment
+    {
+        Top,
+        Center,
+        Bottom
+    }
+    
+    public interface IChartboostMediationBannerView
+    {
+        abstract Dictionary<string, string> Keywords { get; set; }
+        abstract ChartboostMediationBannerAdLoadRequest Request { get; }
+        abstract BidInfo WinningBidInfo { get; }
+        abstract string LoadId { get; }
+        abstract ChartboostMediationBannerAdSize AdSize { get; }
+        abstract ChartboostMediationBannerHorizontalAlignment HorizontalAlignment { get; set; }
+        abstract ChartboostMediationBannerVerticalAlignment VerticalAlignment { get; set; }
+        abstract Task<ChartboostMediationBannerAdLoadResult> Load(ChartboostMediationBannerAdLoadRequest request, ChartboostMediationBannerAdScreenLocation screenLocation);
+        abstract Task<ChartboostMediationBannerAdLoadResult> Load(ChartboostMediationBannerAdLoadRequest request, float x, float y);
+        abstract void SetDraggability(bool canDrag);
```

Comment:
Should this be -> `SetDraggable`?

Reply:
The other function we have is`SetVisibility` so I kept it as `SetDraggability`